Susi/Controllers/SkillDetailViewController/SkillDetailVCMethods.swift

```diff
@@ -479,5 +479,85 @@ extension SkillDetailViewController: UITableViewDelegate, UITableViewDataSource
             }
         }
     }
+    
+    //MARK:- Bookmark skill
+    
+    func showBookmarkButton() {
+        let delegate = UIApplication.shared.delegate as? AppDelegate
+        let user = delegate?.currentUser
```

Comment:
Use a guard statement here instead for the check.

Reply:
Done @chashmeetsingh